src/resmom/linux/mom_mach.h

```diff
@@ -76,6 +76,8 @@ typedef unsigned long ulong;
 #include <basil.h>
 #endif /* MOM_ALPS */
 
+#include "job.h"
```

Comment:
if there were declarations missing from job.h I think it'd show up for other platforms as well. Can you please clarify why you need this include?

Reply:
The `job.h` was needed for the `job` struct typedef. Taking a look at the files that include `mom_mach.h`, they seem to include `job.h` themselves before including `mom_mach.h` so the issue never surfaced. However, `mom_mach.h`'s dependencies should be self-contained so I believe this is correct.

Reply:
ok, thanks for clarifying. I'm a bit wary of headers including other headers because it can result in circular dependencies down the road, but this seems safe enough.